packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -291,11 +291,6 @@ export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
       .next(() => results);
   }
 
-  /**
-   * Returns the set of documents that have been updated since the specified read
-   * time.
-   */
-  // PORTING NOTE: This is only used for multi-tab synchronization.
```

Comment:
Drive by comment: don't the comments still apply? (i.e. isn't this still only used for multi-tab?)

Reply:
The comment is already on the interface declaration.